lesson05/circle_test.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"math"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAreaC(t *testing.T) {
+
+	tests := []struct {
+		name    string
+		args    Circle
+		want    float64
+		wantErr bool
+	}{
+		{
+			name: "if radius is <= 0 should return error ",
```

Comment:
You have a test case for 0 and positive radius. What about negative value? You should probably add a test case for that as well

Reply:
Thanks a lot for usefull review. I have tried all to fix.